File: itinerary.py
itinerary = []

def add_activity(name, hours):
    itinerary.append({"nome": name, "horario": hours})

def remove_activity(name):
    global itinerary
    itinerary = [activity for activity in itinerary if activity["nome"] != name]

def show_itinerary():
    if not itinerary:
        print("\nNenhuma atividade no itinerário.\n")
    else:
        print("\nItinerário de Viagem:")
        for activity in itinerary:
            print(f"- {activity['nome']} às {activity['horario']}")

File: itinerary.dart
List<Map<String, dynamic>> itinerary = [];

void addActivity(String name, String hours) {
  itinerary.add({"nome": name, "horario": hours});
}

void removeActivity(String nome) {
  itinerary.removeWhere((activity) => activity["nome"] == name);
}

void showItinerary() {
  if (itinerary.isEmpty) {
    print("Nenhuma atividade no itinerário.");
  } else {
    print("Itinerário de Viagem:");
    for (var activity in itinerary) {
      print("- ${activity["nome"]} às ${activity["horario"]}");
    }
  }
}
